includes/Modules/Search_Console.php

```diff
@@ -371,21 +373,42 @@ protected function create_search_analytics_data_request( array $args = array() )
 		if ( ! empty( $args['end_date'] ) ) {
 			$request->setEndDate( $args['end_date'] );
 		}
+
+		$filters = array();
+
+		// If domain property, limit data to URLs that are part of the current site.
+		if ( 0 === strpos( $property_id, 'sc-domain:' ) ) {
```

Comment:
I've noticed we have a number of similar conditionals in this module. I think it would make sense to extract this to a method or maybe even a `Search_Console_Property` class which could have a `get_type` method in addition to the `get_id` method. We can revisit this with a dedicated issue; it's not blocking here.